generative_data_prep/utils/utils.py

```diff
@@ -142,6 +142,23 @@ def data_prep_arg_builder(parser: argparse.ArgumentParser):
         help=
         "keyword used in input json to specify completion, defaults to 'completion"
     )
+
+    parser.add_argument(
+        "--prompt_prefix",
+        default=None,
```

Comment:
I would recommend making the default:  ""

Reply:
I would prefer keeping it None just to make it explicitly clear that we do not add anything